getting_started.md

```diff
@@ -28,9 +30,19 @@ Running on the Ubuntu server are:
    # Password: Cisco1234!
    ssh telemetry@10.10.20.25
    ```
-2. Start the Docker Stacks  
+
+2. Download the latest code for the project
+
    ```bash
    cd telemetry_stacks
+   git pull
```

Comment:
Just double checking - it's pre-provisioned with this cloned?

Reply:
It is pre-provisioned but if we ever make changes in the future, this way we don't need to go into the box